src/pkg/build/assembler_sif_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package build
+
+import (
+	"testing"
+
+	"github.com/singularityware/singularity/src/pkg/sylog"
+)
+
+// TestChef sees if we can build a SIF image from a docker://ubuntu:18.04 based kitchen to /tmp
+func TestAssembler(t *testing.T) {
+	def, err := NewDefinitionFromURI("docker://ubuntu:18.04")
```

Comment:
Can you make the test docker URI a const, so it's easy to see at the top. I suspect there will be more tests added, and that would make it visible, and a single place to change.